src/main/java/de/cerus/jdasc/JDASlashCommands.java

```diff
@@ -257,20 +278,32 @@ private JDASlashCommands() {
     public static CompletableFuture<Long> submitGuildCommand(final ApplicationCommand command,
                                                              final long guildId,
                                                              final ApplicationCommandListener listener) {
-        return discordHttpClient.submitGuildCommand(command, guildId).thenApply(response -> {
-            final JsonObject object;
-            try {
-                object = JsonParser.parseString(response.body().string()).getAsJsonObject();
-            } catch (final IOException e) {
-                throw new CompletionException(e);
+        try {
+            List<ApplicationCommand> guildCommands = getGuildCommands(guildId).get();
+            if (guildCommands.contains(command)) {
+                return getLongCompletableFuture(command, listener, guildCommands);
```

Comment:
Could be replaced by [CompletableFuture.completedFuture()](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html#completedFuture-U-)

Reply:
sure, go for it